### ISSUE TITLE
'numpy.ndarray' object has no attribute 'getA1'

### ISSUE THREAD

**Issue by pzq25** [bmabey/pyLDAvis#175]
pyLDAvis.sklearn.prepare error

**Comment by evertorres** [comment]
I changed this two lines in lda_model.py. Seems working 

`## https://numpy.org/doc/stable/reference/generated/numpy.matrix.getA1.html
def _get_doc_lengths(dtm):
#   return dtm.sum(axis=1).getA1()
    return dtm.sum(axis=1).ravel()

def _get_term_freqs(dtm):
   #return dtm.sum(axis=0).getA1()
    return dtm.sum(axis=0).ravel()`
